projects/plugins/jetpack/tests/e2e/package.json

```diff
@@ -13,12 +13,14 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"clean": "rm -rf output allure-results",
+		"clean": "rm -rf output",
 		"distclean": "rm -rf node_modules && yarn clean",
-		"test-e2e-env": "./bin/env.sh start",
-		"test-e2e-env-reset": "./bin/env.sh reset",
-		"pretest-e2e": "yarn clean",
-		"test-e2e": "NODE_CONFIG_DIR='./config' jest --config jest.config.js --runInBand --verbose --detectOpenHandles",
+		"env-start": "./bin/env.sh start",
+		"env-reset": "./bin/env.sh reset",
+		"tunnel-on": "pm2 start bin/ecosystem.config.js",
```

Comment:
Very good point. `tunnel.js` outputs the tunnel info but it gets swallowed by pm2. 
I changed it so `yarn tunnel-on` prints the last few lines of the log file. That should be enough to give relevant info. Also helpful in case of error in `tunnel.js`.

Reply:
Here's what I get:

```
$ /Users/brbrr/Developer/a8c/jetpack/projects/plugins/jetpack/tests/e2e/node_modules/.bin/pm2 logs --nostream --lines 4
[TAILING] Tailing last 4 lines for [all] processes (change the value with --lines option)
/Users/brbrr/.pm2/pm2.log last 4 lines:
PM2        | 2021-04-14T12:51:38: PM2 log: SIGTERM timeout      : 1600
PM2        | 2021-04-14T12:51:38: PM2 log: ===============================================================================
PM2        | 2021-04-14T12:51:38: PM2 log: App [tunnel:0] starting in -fork mode-
PM2        | 2021-04-14T12:51:40: PM2 log: App [tunnel:0] online

/Users/brbrr/.pm2/logs/tunnel-error.log last 4 lines:
0|tunnel   | 2021-04-12T18:45:30: Tunnels file doesn't exist
0|tunnel   | 2021-04-14T12:51:39: WARNING: NODE_APP_INSTANCE value of '0' did not match any instance config file names.
0|tunnel   | 2021-04-14T12:51:39: WARNING: See https://github.com/lorenwest/node-config/wiki/Strict-Mode
0|tunnel   | 2021-04-14T12:51:39: Request failed with status code 404

/Users/brbrr/.pm2/logs/tunnel-out.log last 4 lines:
0|tunnel   | 2021-04-14T12:51:39: Checking if tunnel for honest-wombat-52 is on
0|tunnel   | 2021-04-14T12:51:39: Tunnel for honest-wombat-52 is OFF (undefined)
0|tunnel   | 2021-04-14T12:51:39: Opening tunnel. Subdomain: 'honest-wombat-52'
0|tunnel   | 2021-04-14T12:51:40: Opened tunnel for 'honest-wombat-52'
```

I don't think logging only a subdomain is enough. Since this is a cli tool, I feel that nice, human-readable, and actionable output is crucial in this case. Having the full URL would make it much easier to open a new browser tab with the tunneled URL (similar to `WARNING` above)

Reply:
> I don't think logging only a subdomain is enough.

I agree. The full URL is now logged.